apps/server/src/routers/healthRouter.ts

```diff
@@ -21,7 +21,7 @@ import { dbHealth, Status } from '../app-health';
 
 const router = Router();
 
-router.get('/', (req, res) => {
+router.get('/', (_req, res) => {
```

Comment:
is this to imply `req` not being used? seems superfluous a change

Reply:
Indicating unused parameter. Gets flagged by TS because of the `noUnusedParamter=true` setting in .tsconfig . Helps indicate to a reader that the argument is a function placeholder and not used. Minor but I find it helpful :)